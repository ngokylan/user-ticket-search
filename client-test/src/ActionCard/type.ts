import { ReactChild, ReactNode } from 'react';
import { ButtonOnClick, ButtonProps } from '../Button';
import { BadgeProps } from '../Badge';

export type ActionCardButtonProp = Pick<
  ButtonProps,
  | 'isDisabled'
  | 'type'
  | 'isUppercase'
  | 'isText'
  | 'isCircular'
  | 'isLoading'
  | 'loadingTitle'
  | 'id'
> & {
  icon: ReactNode;
  onClick: ButtonOnClick;
  label: string;
};

export type StatusBadgeProp = Pick<BadgeProps, 'type' | 'icon'> & {
  label: string;
};
export type StatusPopoverBadgeProp = Pick<BadgeProps, 'type' | 'icon'> & {
  label: string;
  content: ReactNode;
};

export type DropdownOption = {
  icon?: ReactChild;
  label: string;
  onClick: () => void;
  isDisabled?: boolean;
  withDivider?: boolean;
};

export type HeaderOptionsCustom = Pick<
  ButtonProps,
  'icon' | 'onClick' | 'ariaLabel' | 'id'
>;

export type HeaderOptionsDropdown = {
  id?: string;
  icon?: ReactNode;
  dropdownOptions: DropdownOption[];
  /** ariaLabel is for WCAG accessibility compliance. Although this is an optional prop,
   * you must use it in order to be WCAG compliant.
   */
  ariaLabel?: string;
};

export type HeaderOptions = HeaderOptionsCustom | HeaderOptionsDropdown;

// Props:
export type ActionCardHeaderProps = {
  icon?: ReactNode;
  title?: string;
  titleAddOn?: string;
  badge?: StatusBadgeProp;
  popoverBadge?: StatusPopoverBadgeProp;
  options?: HeaderOptions;
};

export type ActionCardHeaderBadgeProps = {
  badge: StatusBadgeProp;
};

export type ActionCardHeaderPopoverBadgeProps = {
  popoverBadge: StatusPopoverBadgeProp;
};

export type ActionCardHeaderOptionsProps = {
  options: HeaderOptions;
};

export type ActionCardContentProps = {
  icon?: ReactNode;
  heading: string;
  headingAddOn?: ReactNode;
  children: ReactNode;
  /**
   * role is for WCAG accessibility compliance, (optional). This prop was added primarily
   * for use within child components that might be used inside the ActionCard component
   * (e.g. the Row and Col children components in the Demo). So unless you have a special reason
   * to use it, you can ignore it.
   */
  role?: string;
};

export type ActionCardFooterProps = {
  actions: ActionCardButtonProp[];
};

export type ActionCardProps = {
  /** ActionCard content (must be ActionCard.Content component) */
  children: ReactNode;
  /** ActionCard id attribute (optional) */
  id?: string;
  /** Action card header (optional) */
  header?: ActionCardHeaderProps;
  /** ActionCard footer buttons (optional) */
  actions?: ActionCardButtonProp[];
  /** ActionCard expand/collapse functionality (optional) */
  isExpandable?: boolean;
  /** expandBtnAriaLabel is for WCAG accessibility compliance, defaults to "Expand" if omitted (optional). */
  expandBtnAriaLabel?: string;
};

export type ActionCardState = {
  isExpanded: boolean;
};
