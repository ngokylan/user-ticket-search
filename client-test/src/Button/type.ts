import { ReactNode, MouseEventHandler, FocusEventHandler } from 'react';
import { BadgeProps } from '../Badge';
export type ButtonType = 'default' | 'primary' | 'danger';
export type ButtonHtmlType = 'submit' | 'button' | 'reset';
export type ButtonOnClick = MouseEventHandler<HTMLButtonElement>;
export type ButtonFocusEventHandler = FocusEventHandler<HTMLButtonElement>;
export type ButtonLabel = ReactNode;

export const ButtonTypes: { [key: string]: ButtonType } = {
  default: 'default',
  primary: 'primary',
  danger: 'danger',
};

export type BadgePropsExtended = string | number | BadgeProps;

export type ButtonBadgeProps = {
  /** id attribute, optional id */
  id?: string;
  /** string | number | BadgeProps */
  badge: BadgePropsExtended;
};

export type ButtonProps = {
  /** Button click callback, triggered when the button is clicked, (optional) */
  onClick?: ButtonOnClick;
  /** Button blur callback, triggered when the button is losing focus, (optional) */
  onBlur?: ButtonFocusEventHandler;
  /** Button focus callback, triggered when the button is focused, (optional) */
  onFocus?: ButtonFocusEventHandler;
  /** Button Enable/Disable status, (optional) */
  isDisabled?: boolean;
  /** Button id attribute, optional id */
  id?: string;
  /** Button custom Icon, (optional) */
  icon?: ReactNode;
  /** Button HTML type, use one from the following type: submit | button | reset, (optional) */
  htmlType?: ButtonHtmlType;
  /** Button css style type, use one from the following css type: default | text | primary | danger, (optional) */
  type?: ButtonType;
  /** Button class name attribute, (optional) */
  className?: string;
  /** Button block status, (optional) */
  isBlock?: boolean;
  /** Button css text transform into Uppercase, (optional) */
  isUppercase?: boolean;
  /** Button css text style, (optional) */
  isText?: boolean;
  /** Button css circular style (optional) */
  isCircular?: boolean;
  /** Button css margin left -12px (optional) */
  isLeftIndent?: boolean;
  /** Button label */
  children?: ReactNode;
  /** To show if Button is in loading status */
  isLoading?: boolean;
  /** Title of button when it is in loading */
  loadingTitle?: string;
  /** To have a badge on top of button */
  badge?: BadgePropsExtended;
  /** Add ariaLabel for WCAG accessibility compliance when button has no text, (optional) */
  ariaLabel?: string;
  /** Role is for WCAG accessibility compliance, defaults to "button" if omitted (optional) */
  role?: string;
  /** ariaChecked is for WCAG accessibility compliance, (optional). This prop was added primarily
   * for use within the Switcher component, so unless you have a special reason to use it, you can
   * ignore it.
   */
  ariaChecked?: string;
  /** ariaHaspopup is for WCAG accessibility compliance, (optional). This prop was added primarily
   * for use within the Dropdown component, so unless you have a special reason to use it, you can
   * ignore it.
   */
  ariaHaspopup?: boolean;
  /** ariaExpanded is for WCAG accessibility compliance, (optional). This prop was added primarily
   * for use within the Dropdown component, so unless you have a special reason to use it, you can
   * ignore it.
   */
  ariaExpanded?: boolean;    
};

export type ButtonStates = {};
