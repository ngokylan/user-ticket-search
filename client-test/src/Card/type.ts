import { ReactNode } from 'react';
import { BadgeProps } from '../Badge';

export interface CardProps {
  /** Card id attribute, optional id */
  id?: string;
  /** Card heading title */
  heading?: ReactNode;
  /** Card description */
  description?: string;
  /** Card content */
  children: ReactNode;
  /** Additional class */
  className?: string;
  /** Make height 100% */
  isFullHeight?: boolean;
  /** To place option content to the right side of heading. such as Button */
  option?: ReactNode;
  /** small heading above heading */
  prefix?: string;
  /** image in the header */
  image?: string;
  /** array of the badge components which are displayed on image */
  badges?: BadgeProps[];
  /** function that is triggered if the card is clicked */
  onClick?: Function;
  /** Role is for WCAG accessibility compliance (optional) */
  role?: string;
}
