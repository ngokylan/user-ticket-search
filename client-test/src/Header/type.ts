import { MouseEventHandler, ReactNode } from 'react';
import { BadgeProps } from '../Badge';

export type HeaderProps = {
  /** The acceptable components are Search and BulkAction only. (optional) */
  children?: ReactNode;
  /**
   * Header action buttons goes next to the Primary button. (optional)
   * Mainly, it is used to show the Search/BulkAction buttons.
   * For more usage, please refer to the template app.
   */
  buttons?: Array<HeaderButtonProp>;
  /**
   * The primary themed button. The icon is optional if it's text only button. (optional)
   * It is shaped as below:
   *
   * PrimaryButtonType = {
   *   icon?: ReactNode;
   *   label: string;
   *   onClick: Function;
   * }
   */
  primaryButton?: PrimaryButtonType;
  /**
   * The back button is rendered with the left arrow icon automatically. (optional)
   *
   * BackButtonType = {
   * title: string;
   * onClick: Function;
   * ariaLabel?: string;
   * }
   */
  backButton?: BackButtonType;
  /** Title of the page. (optional) */
  title?: ReactNode;
  /**
   * Any nodes that goes to the end of title. (optional)
   * For now, the Badge is the only case.
   */
  titleAddon?: ReactNode;
  /** The Sub Heading goes under the main title. (optional)  */
  subHeading?: string;
  /**
   * To switch Header and BulkAction.
   * Please use it with Layout Context so it is also synced with the List Table. (optional)
   * For more usage, please refer to the template app.
   */
  isBulkActionOpen?: boolean;
  /**
   * To switch Header and Search. Please use it with Layout Context. (optional)
   * For more usage, please refer to the template app.
   */
  isSearchOpen?: boolean;
  /**
   * The second level navigation is rendered at the bottom of Header. (optional)
   * The array of these props.
   *
   * HeaderTabLinkProps = {
   * active: boolean;
   * label: string;
   * onClick: Function;
   * }
   */
  tabs?: Array<HeaderTabLinkProps>;
  theme?: Theme;
};

export type BackButtonType = {
  title: string;
  onClick: Function;
  /** Add ariaLabel for WCAG accessibility compliance. Although optional, you
   * must add an ariaLabel for full WCAG compliance.
   */
  ariaLabel?: string;
};

export type PrimaryButtonType = PrimaryButtonIconType | PrimaryButtonLabelType;
export type HeaderBadgePropsExtended = string | number | BadgeProps;

export type PrimaryButtonCoreType = {
  onClick?: MouseEventHandler<HTMLButtonElement>;
  children?: Array<HeaderButtonProp>;
};

export type PrimaryButtonIconType = PrimaryButtonCoreType & {
  icon: ReactNode;
  label?: string;
  isDisabled?: boolean;
  badge?: HeaderBadgePropsExtended;
  /**
   * Add `ariaLabel` if `label` is empty or doesn't exist. For WCAG accessibility compliance.
   * Although optional, you must add an ariaLabel for full WCAG compliance.
   */
  ariaLabel?: string;
};

export type PrimaryButtonLabelType = PrimaryButtonCoreType & {
  icon?: ReactNode;
  label: string;
  isDisabled?: boolean;
  badge?: HeaderBadgePropsExtended;
  /**
   * Add `ariaLabel` if `label` is empty or doesn't exist or you want to have different `label`
   * and `ariaLabel` values. For WCAG accessibility compliance. Although optional, you must add
   * an ariaLabel for full WCAG compliance.
   */
  ariaLabel?: string;
};

export type HeaderButtonProp = {
  icon: ReactNode;
  label: string;
  onClick: Function;
  children?: Array<HeaderButtonProp>;
  hasDivider?: boolean;
  isDisabled?: boolean;
  badge?: HeaderBadgePropsExtended;
  /**
   * Add `ariaLabel` if `label` is empty or doesn't exist or you want to have different `label`
   * and `ariaLabel` values. For WCAG accessibility compliance. Although optional, you must add
   * an ariaLabel for full WCAG compliance.
   */
  ariaLabel?: string;
};

export type HeaderButtonsProps = {
  children?: ReactNode;
  buttons?: Array<HeaderButtonProp>;
  primaryButton?: PrimaryButtonType;
};

export type HeaderBackProps = {
  title?: string;
  onClick: Function;
  /** Add ariaLabel for WCAG accessibility compliance. Defaults to
   * 'Back to previous screen' if omitted.
   */
  ariaLabel?: string;
};

export type HeaderDetailsItemProps = {
  icon: ReactNode;
  content: any;
};

export type HeaderDetailsProps = {
  details: Array<HeaderDetailsItemProps>;
};

export type HeaderTabLinkProps = {
  active: boolean;
  label: string;
  onClick: MouseEventHandler<HTMLAnchorElement>;
};

export type HeaderTabsProps = {
  tabs: Array<HeaderTabLinkProps>;
};

export type HeaderTitleProps = {
  children?: ReactNode;
  subHeading?: string;
};

export type Theme = {
  primaryColor?: string;
};
