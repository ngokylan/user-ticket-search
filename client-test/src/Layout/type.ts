import { ReactNode } from 'react';
import { MENU_STATE_TYPE_MIN, MENU_STATE_TYPE_WIDE } from '../_lib/const';

export type LayoutProps = {
  children?: ReactNode;
  /** To attach Menu Component */
  menu?: ReactNode;
  /** Use Reboot.scss */
  isStyleReboot?: boolean;
  theme?: LayoutTheme;
};

export type LayoutTheme = 'min' | 'default';
export const LayoutThemes: { [key: string]: LayoutTheme } = {
  default: 'default',
  min: 'min',
};

export type MenuStateType = MENU_STATE_TYPE_MIN | MENU_STATE_TYPE_WIDE;
export type LayoutContextProps = {
  /** Menu staus callback method */
  setMenuStatus: (menuState: MenuStateType) => void;
  /** Menu state, min | wide */
  menuState: MenuStateType;
  /** To allow content to scroll */
  isContentScrolling: boolean;
  /** Side panel state */
  isSidePanelOpen: boolean;
  /** Display side panel overlay */
  isSidePanelOverlay: boolean;
  /** Display search */
  isSearchOpen: boolean;
  /** Display side panel overlay */
  toggleSidePanelOverlay: any;
  /** Toggle search */
  toggleSearchOpen: any;  
  /** To pass layoutRef within the context */
  layoutRef?: Element;
  /** A function that receives ref and set context in layoutRef */
  setLayoutRef: Function;
  /** To pass headerRef within the context */
  headerRef?: Element;
  /** A function that receives ref and set context in headerRef */
  setHeaderRef: Function;
};
