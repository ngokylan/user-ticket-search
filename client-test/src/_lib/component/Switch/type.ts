import { ReactNode, MouseEventHandler, ChangeEventHandler } from 'react';

export type SwitchOnClick = MouseEventHandler<HTMLInputElement>;
export type SwitchOnChange = ChangeEventHandler<HTMLInputElement>;

export type SwitchProps = {
  onClick?: SwitchOnClick;
  onChange?: SwitchOnChange;
  id: string;
  isOn: boolean;
  ariaLabel?: string;
};
