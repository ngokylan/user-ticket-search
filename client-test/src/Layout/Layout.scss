@import '../_lib/style/index';
$menu-width-min-close: 56px;
$menu-width-max-close: 272px;
$menu-width-min-max: 328px;
$menu-width-max-max: 544px;
$menu-height-mobile: 50px;
$Layout: '.layout';

body {
  background-color: $gray-200;
}

#{$Layout} {
  display: flex;
  /* Layout structure */
  flex-direction: column;
  min-height: 100vh;

  @include desktop-lg {
    flex-direction: row;
  }

  &__main-menu {
    flex-shrink: 0;
    flex-grow: 0;
  }

  /* Small Screen - Main menu goes on top */
  @include desktop-max {
    #{$Layout}__main-app {
      box-shadow: none;
      overflow: visible;
      flex-grow: 1;

      &:after {
        content: '';
        position: absolute;
        display: block;
        background-color: rgba(255, 255, 255, 0.5);
        width: 100%;
        height: 100%;
        left: 0;
        top: 0;
        z-index: -1;
        opacity: 0;
      }
    }

    &.menu-status-open-mobile {
      #{$Layout}__main-app {
        overflow: hidden;

        &:after {
          content: '';
          opacity: 1;
          z-index: 9;
        }
      }

      #{$Layout}__main-content,
      .header {
        transform: translateX(#{$menu-width-max-close});
      }
    }

    #{$Layout}__main-menu {
      position: fixed;
      top: 0;
      left: 0;
      width: 100%;
      z-index: 5;
    }
  }
}

$MinHeader: 80px;
$MaxHeader: 224px;

#{$Layout}__main-app {
  position: relative;
  width: 100%;
  overflow: hidden;
  transition: padding-right 0.3s;
  display: flex;
  flex-direction: column;
  background-color: $gray-200;

  @include desktop-lg {
    height: 100vh;
  }
}

#{$Layout}__main-header {
  position: relative;
  width: 100%;
}

#{$Layout}__main-content-wrapper {
  position: relative;
  overflow-y: scroll;
  scroll-behavior: smooth;
  -webkit-overflow-scrolling: touch;
  height: 100%;

  @include desktop-max {
    height: auto;
    overflow-y: visible;
    flex-grow: 1;
    display: flex;
    width: 100%;
  }
}

#{$Layout} {
  &--is-sidepanel-open {
    #{$Layout}__main-app {
      padding-right: $SidePanelWidthDesktop;

      @include tablet-max {
        padding-right: 0;
      }
    }

    #{$Layout}__main-menu {
      @include desktop-max {
        @include desktop {
          width: calc(100% - 344px);
        }
      }
    }
  }

  &--is-modal-open {
    #{$Layout}__main-content-wrapper {
      overflow-y: visible;
    }
  }
}

@keyframes ani-side-panel {
  from {
    transform: translateX(100%);
  }

  to {
    transform: translateX(0);
  }
}

body {
  &.scroll-hidden {
    overflow: hidden;
  }
}

/* Temporary Left Nav */

.tmp-nav-content {
  transition: all 0.3s;
  height: 50px;

  button {
    display: none;
    width: 40px;
    height: 40px;
    margin: 5px;
  }

  @include desktop-lg {
    height: 100%;
    overflow-y: auto;

    button {
      display: inline-block;
    }
  }
}

@include desktop-lg {
  .layout.menu-status-wide {
    .tmp-nav-content {
      width: 272px;
    }
  }

  .layout.menu-status-min {
    .tmp-nav-content {
      width: 56px;
    }
  }
}

.layout--theme-min {
  // Note that Header-related styles is in Header.scss.
  &#{$Layout} {
    #{$Layout}__main-app {
      box-shadow: none;
      overflow: visible;
      flex-grow: 1;

      &:after {
        content: '';
        position: absolute;
        display: block;
        background-color: rgba(255, 255, 255, 0.5);
        width: 100%;
        height: 100%;
        left: 0;
        top: 0;
        z-index: -1;
        opacity: 0;
      }
    }

    &.menu-status-open-mobile {
      #{$Layout}__main-app {
        overflow: hidden;

        &:after {
          content: '';
          opacity: 1;
          z-index: 9;
        }
      }

      #{$Layout}__main-content,
      .header {
        transform: translateX(#{$menu-width-max-close});
      }
    }

    #{$Layout}__main-menu {
      position: fixed;
      top: 0;
      left: 0;
      width: 100%;
      z-index: 5;
    }
  }

  #{$Layout}__main-app {
    height: auto;
  }

  #{$Layout}__main-content-wrapper {
    height: auto;
    overflow-y: visible;
    flex-grow: 1;
    display: flex;
    width: 100%;
  }
}
