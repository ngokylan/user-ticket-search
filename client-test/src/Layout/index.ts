export { default } from './Layout';
export { default as withLayoutAware } from './component/LayoutAware';
export * from './type';
