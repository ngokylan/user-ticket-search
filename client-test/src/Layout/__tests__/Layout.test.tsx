import React from 'react';
import { mount } from 'enzyme';
import Layout from '../';
import { LayoutProps } from '../type';

const defaultProps: LayoutProps = {
  isStyleReboot: true,
  theme: 'default',
};
const mountComponent = (customProps: any = {}) => {
  const props = { ...defaultProps, ...customProps };
  const wrapper = mount(<Layout {...props} />);
  const component = wrapper.find('.layout__main-app');

  return { props, wrapper, component };
};

describe('Layout', () => {
  describe('rendering', () => {
    it('should render component without any crash', () => {
      const { wrapper } = mountComponent();
      expect(wrapper).toHaveLength(1);
    });

    it('should be match with snapshot', () => {
      const { wrapper } = mountComponent();
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('props', () => {
    describe('default', () => {
      it('should have expected default props', () => {
        expect(JSON.stringify(defaultProps)).toBe(
          JSON.stringify(Layout.defaultProps)
        );
      });
    });

    describe('children', () => {
      it('should render children', () => {
        const text = 'Hello world';
        const { component } = mountComponent({ children: text });
        expect(component.text()).toBe(text);
      });
    });
  });

  describe('interaction', () => {
    it.each([
      ['should', '.layout', 1, {}],
      ['should', '.elements', 1, {}],
      ['should not', '.layout--is-search-open', 0, {}],
    ])('%s have %s', (name, element, expected, props) => {
      const { wrapper } = mountComponent(props);
      expect(wrapper.find(element)).toHaveLength(expected);
    });
  });

  describe('api', () => {
    it('should call setState properly when toggleSearchOpen is called', () => {
      jest.clearAllMocks();
      const { wrapper } = mountComponent();
      const checkbox = wrapper.instance();

      checkbox.toggleSearchOpen();
      expect(wrapper.state('isSearchOpen')).toEqual(true);
      checkbox.toggleSearchOpen();
      expect(wrapper.state('isSearchOpen')).toEqual(false);
    });

    it('should call setState properly when toggleSidePanelOverlay is called', () => {
      jest.clearAllMocks();
      const { wrapper } = mountComponent();
      const checkbox = wrapper.instance();

      checkbox.toggleSidePanelOverlay();
      expect(wrapper.state('isSidePanelOverlay')).toEqual(true);
      checkbox.toggleSidePanelOverlay();
      expect(wrapper.state('isSidePanelOverlay')).toEqual(false);
    });

    it('should call setState properly when setMenuStatus is called', () => {
      jest.clearAllMocks();
      const { wrapper } = mountComponent();
      const checkbox = wrapper.instance();

      checkbox.setMenuStatus('min');
      expect(wrapper.state('menuState')).toEqual('min');
      checkbox.setMenuStatus('wide');
      expect(wrapper.state('menuState')).toEqual('wide');
    });

    it('should set headerRef when setHeaderRef is called and dispatch event', () => {
      jest.clearAllMocks();
      const onUpdateHeaderRef = jest.fn();
      global.addEventListener('onUpdateHeaderRef', onUpdateHeaderRef, true);

      const { wrapper } = mountComponent();
      const componentInstance = wrapper.instance();
      const testRef = 'testRef';
      componentInstance.setHeaderRef(testRef);
      expect(componentInstance.state.headerRef).toBe(testRef);
      expect(onUpdateHeaderRef).toBeCalledTimes(1);
    });

    it('should set layoutRef when setLayoutRef is called and dispatch event', () => {
      jest.clearAllMocks();
      const onUpdateLayoutRef = jest.fn();
      global.addEventListener('onUpdateLayoutRef', onUpdateLayoutRef, true);

      const { wrapper } = mountComponent();
      const componentInstance = wrapper.instance();
      const testRef = 'layoutRef';
      componentInstance.setLayoutRef(testRef);
      expect(componentInstance.state.layoutRef).toBe(testRef);
      expect(onUpdateLayoutRef).toBeCalledTimes(1);
    });
  });
});
