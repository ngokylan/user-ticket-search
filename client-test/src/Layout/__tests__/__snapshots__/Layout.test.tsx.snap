// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout rendering should be match with snapshot 1`] = `
<div
  className="layout menu-status-wide elements layout--theme-default"
>
  <div
    className="layout__main-app"
  />
</div>
`;
